CHANGELOG.md

```diff
@@ -53,11 +53,11 @@ These functions and structs can be referenced in both code and blueprints it may
 - Handover is now optional depending on whether the load balancing strategy implementations require it . See `RequiresHandoverData`
 - Improved the failed hierarchy migration logs. The logs now contain more specific reasons for the failure and the frequency of repeated logs is suppressed.
 - SpatialWorldSettings is now the default world settings in supported engine versions.
-- Worker SDK version compatibility is checked at compile time. 
```

Comment:
very minor but this is unneeded churn

Reply:
Ty, will push along with a fix for the build :)